packages/types/src/server.ts

```diff
@@ -14,6 +14,8 @@ export interface LaunchArgs {
   cwd: string
   browser: Browser
   configFile?: string
+  // Global mode is triggered by CLI via `--global` or when there is no `projectRoot` (essentially when the Cypress Config file can't be found)
```

Comment:
I don't think this should be nullable - either `true` or `false` (making in nullable doesn't really add any value. We'd assume `undefined` is `false`, so I think it's better be explicit.